source/plugin/bookview.rst

```diff
@@ -0,0 +1,36 @@
+==========
+Book Views
+==========
+
+A ``BookView`` is the displaying of the Book GUI on the client. The ``BookView`` is not associated with an actual
```

Comment:
First sentence is a little awkwardly worded. Maybe "is the representation of the Book GUI..."